scripts/test_env.py

```diff
@@ -118,18 +118,19 @@ def run(self, is_genesis: bool = False):
 
         os.environ["KURA_BLOCK_STORE_PATH"] = str(peer_dir.joinpath("storage"))
         os.environ["SNAPSHOT_DIR_PATH"] = str(peer_dir.joinpath("storage"))
-        os.environ["LOG_FILE_PATH"] = str(peer_dir.joinpath("log.json"))
-        os.environ["MAX_LOG_LEVEL"] = "TRACE"
+        os.environ["LOG_LEVEL"] = "TRACE"
+        # FIXME: should it be JSON here?
```

Comment:
Not sure on this one, for quick debugging at least for me it's more convenient to read plain text.

Reply:
I think it's better to use `pretty` in this case.

Reply:
I _think_ with bunyan format we had before this PR it was possible to use a [cli tool](https://www.npmjs.com/package/bunyan) to print it in a pretty form with colors, which is nice. I learned of it just now, after researching what is this format :sweat_smile: